composer.json

```
@@ -24,11 +24,11 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^5.6 || ^7.0",
-        "squizlabs/php_codesniffer": "^2.6"
+        "php": "^7.0 || ^8.0",
+        "squizlabs/php_codesniffer": "3.*"
```

Please use the caret syntax as the preferred symantic versioning.
```suggestion
        "squizlabs/php_codesniffer": "^3.0"
```

---

Additionally it looks like upgrading to codesniffer 3 has a lot of backward breaking changes to custom sniffs. See here: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Version-3.0-Upgrade-Guide